### PR TITLE
test(dm): align TiDB SHOW statement output

### PR DESCRIPTION
**Pull request by lance6716**
Signed-off-by: lance6716 <lance6716@gmail.com>

### What problem does this PR solve?

Issue Number: ref #4159

### What is changed and how it works?

align with https://github.com/pingcap/tidb/pull/35136

#### Tests 

 - Integration test

### Release note 

```release-note
 `None`.
```

### CODE REVIEW

**Review by Ehco1996 — APPROVED** [review]

File: dm/tests/full_mode/run.sh [hunk 1]
```diff
@@ -150,7 +150,7 @@ function only_route_schema() {
 		"query-status test" \
 		"\"stage\": \"Finished\"" 1
 
-	run_sql_tidb_with_retry "SHOW DATABASES LIKE 'full_mode_test';" "Database: full_mode_test"
+	run_sql_tidb_with_retry "SHOW DATABASES LIKE 'full_mode_test';" "Database (full_mode_test): full_mode_test"
```
Comment (D3Hunter) [inline dm/tests/full_mode/run.sh:153]:
```suggestion
	run_sql_tidb_with_retry "SHOW DATABASES LIKE 'full_mode_test';" ": full_mode_test"
```
maybe this way, we don't care about heading here.

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**